docs/content/guides/dagster/migrating-to-pythonic-resources-and-config.mdx

```diff
@@ -0,0 +1,313 @@
+---
+title: Migrating to Pythonic resources and config | Dagster
+description: A guide to incrementally migrating existing Dagster codebases to Pythonic resources and config
+---
+
+# Migrating to Pythonic Resources and Config
+
+With Dagster 1.3 we added a new set of APIs to our resources and config based on Pydantic, rather than our own bespoke configuration schema API. This added more type safety and made the Python APIs more idiomatic. This guide provides information on how to migrate from our legacy resource and config system to this new system, even in large existing Dagster codebases.
```

Comment:
```suggestion
With Dagster 1.3 we added a new set of APIs to our resources and config systems based on Pydantic, rather than our own bespoke configuration schema API. This added more type safety and made the Python APIs more idiomatic. This guide provides information on how to migrate from our legacy resource and config system to this new system, even in large existing Dagster codebases.
```
nit - could be a word other than systems too